jerry-core/api/jerry-debugger.c

```diff
@@ -116,3 +116,63 @@ jerry_debugger_cleanup (void)
   }
 #endif /* JERRY_DEBUGGER */
 } /* jerry_debugger_cleanup */
+
+/**
+ * Sets whether the engine should wait for a source.
+ */
+void
```

Comment:
Could be `bool` or `jerry_value_t` to tell about what happened (e.g. the returned value of the jerry_run on success, some error otherwise).